src/python/director/cameraview.py

```diff
@@ -385,44 +385,77 @@ def updateView(self):
 
 class ImageWidget(object):
 
-    def __init__(self, imageManager, imageName, view):
+    def __init__(self, imageManager, imageName, view, visible=True):
         self.view = view
         self.imageManager = imageManager
         self.imageName = imageName
+        self.visible = visible
 
         self.updateUtime = 0
         self.initialized = False
 
         self.imageWidget = vtk.vtkLogoWidget()
-        rep = self.imageWidget.GetRepresentation()
-        rep.GetImageProperty().SetOpacity(1.0)
+        imageRep = self.imageWidget.GetRepresentation()
+        self.imageWidget.ResizableOff()
+        self.imageWidget.SelectableOn()
+
+        imageRep.GetImageProperty().SetOpacity(1.0)
         self.imageWidget.SetInteractor(self.view.renderWindow().GetInteractor())
 
         self.flip = vtk.vtkImageFlip()
         self.flip.SetFilteredAxis(1)
         self.flip.SetInput(imageManager.getImage(imageName))
-        rep.SetImage(self.flip.GetOutput())
+        imageRep.SetImage(self.flip.GetOutput())
 
         self.timerCallback = TimerCallback()
         self.timerCallback.targetFps = 60
         self.timerCallback.callback = self.updateView
         self.timerCallback.start()
 
+        self.initialized = False
+
+    def setWidgetSize(self, desiredWidth=400):
+        viewWidth, viewHeight = self.view.width, self.view.height
+        defaultImageAspectRatio = 4/3.
+        imageWidth, imageHeight = desiredWidth, desiredWidth/defaultImageAspectRatio
+
+        rep = self.imageWidget.GetBorderRepresentation()
+        rep.SetShowBorderToOff()
+        coord = rep.GetPositionCoordinate()
+        coord2 = rep.GetPosition2Coordinate()
+        coord.SetCoordinateSystemToDisplay()
+        coord2.SetCoordinateSystemToDisplay()
+        coord.SetValue(0, viewHeight-imageHeight)
+        coord2.SetValue(imageWidth, imageHeight)
+
+        self.view.render()
+
+    def setImage(self, imageName):
+        self.imageName = imageName
+        self.flip.SetInput(imageManager.getImage(imageName))
```

Comment:
i think this method setImage(imageName) could be removed, no need to support this api.  is anyone using this method?  or keep it but change the function name to setImageName()